### PR TITLE
PR for Cart page

### PR DESCRIPTION
**Pull request by kiranpratyush**
Added Cart page to my eCommerce site. Please review and give feedback(Note: This site is not responsive yet)
[Testing link](https://testing2322.netlify.app/pages/cart/index.html)

### CODE REVIEW

File: pages/cart/style.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+main {
+  max-width: 1500px;
+  display: flex;
+  padding-left: 100px;
+  padding-right: 100px;
```
Comment (bipin7yadav) [inline pages/cart/style.css:5]: you can use rem instead of pixel

Comment (manojkumar8400) [inline pages/cart/style.css:5]: suggestion -: you can use rem instead of px.

File: pages/cart/style.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  gap: 50px;
+}
+
+
+.cart-total {
+  align-self: center;
+  max-width: 100%;
+  height: 200px;
+  flex-grow: 1;
+  max-width: 800px;
```
Comment (manojkumar8400) [inline pages/cart/style.css:26]: Suggestion -: You have don't use the same property twice and use rem,vw,% instead of px in width.

Comment (bipin7yadav) [inline pages/cart/style.css:26]: don't use same property twice

**Review by bipin7yadav — APPROVED** [review]

**Review by manojkumar8400 — APPROVED** [review]

File: pages/cart/style.css (hunk 1 above)
Comment (kiranpratyush) [reply pages/cart/style.css:5]: Thank you I'll change it

Comment (kiranpratyush) [reply pages/cart/style.css:5]: Thank you I'll change it

File: pages/cart/style.css (hunk 2 above)
Comment (kiranpratyush) [reply pages/cart/style.css:26]: Thank you I'll change it

Comment (kiranpratyush) [reply pages/cart/style.css:26]: Thank you I'll change it

**Pull request merged**